src/index.ts

```diff
@@ -814,7 +847,7 @@ export class AcrolinxEndpoint {
         throw error;
       }
     } else {
-      return fetchFunction(input, fetchProps);
+      return fetch(input, fetchProps);
```

Comment:
@JustinHorn It is important we return `fetchFunction` here. The SDK is intended to work for both browser as well as Node. Node applications set their own `fetch` function, as they have to deal with proxies, and different types of network authentications

Reply:
ah yea, replacing that was accidental